model/registry/rule.yaml

```diff
@@ -0,0 +1,67 @@
+groups:
+  - id: registry.rule
+    prefix: rule
+    type: attribute_group
+    brief: >
+      Describes rule attributes. Rule fields are used to capture the specifics of any observer or agent rules
+      that generate alerts or other notable events.
+    attributes:
```

Comment:
I'm coming from a background of working closely with the [Sigma project](https://github.com/SigmaHQ/Sigma), and they have a couple of metadata fields in their [rule specification](https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md) that we've made good use of (specifically the `level` attribute that conveys the severity of a rule being triggered and the ability to tag rules with values that associate it with particular stages in attacker ontologies, such as [MITRE ATT&CK](https://attack.mitre.org/#)). I was wondering if there was any scope for expanding the attribute list, and if so, whether either of those would be suitable?

Reply:
hey @kelnage, thanks for checking out this PR. I don't see any objections on expanding the rule nameset. But I would say that it either needs to be generic or we need to discuss how to add fields specific for application. Maybe it should be something like `rule.security.level` or maybe `rule.level` would be appropriate enough. I suggest to create an issue or PR as soon as this PR is merged to discuss proposed changes